docs/dg/projectManagement.md

```diff
@@ -21,6 +21,27 @@ This page contains information about project management tasks. The target audien
 
 <!-- ==================================================================================================== -->
 
+## Making a release on Github
+
+Before making a release, please check the following prerequisites:
+
+* Ensure that you have **JDK `1.8.0`** installed (==Not other major release versions such as **JDK `9`** or **JDK `11`**==).
+* Ensure that the `JAVA_HOME` environment variable is correctly set to your JDK installation directory. You can refer to the [JDK Installation Guide](https://docs.oracle.com/cd/E19182-01/821-0917/inst_jdk_javahome_t/index.html).
+
+To make a release for RepoSense on Github, please follow the `Creating a release` section in the [Github Docs](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).<br>
+  
+Take note of the following when making the release according to the above guide:
+* When entering a release version number, use semantic versioning with some small tweaks:
+  * Use `MAJOR.MINOR` as the version number when the release includes new features and/or major changes.
+  * Use `MAJOR.MINOR.PATCH` as the version number when the release only includes bug fixes and/or minor changes.
+  * Append `rc` to the version number to indicate that the release is a pre-release that is not ready to be used in production.
+* Enter the release title as `RepoSense vxxx` where `xxx` is the version number. Enter the release description by referring to the previous [RepoSense releases](https://github.com/reposense/RepoSense/releases). 
+* Before launching the release, generate the `RepoSense.jar` file and attach it to the release.
+  * Change the directory to the project root directory.
```

Comment:
Maybe can additional steps to check the Java version here by `gradlew --version`?

Reply:
Or add this check after line 29?

Reply:
Sure, no problem